### PR TITLE
Update docs for 0.11-M4

### PR DESCRIPTION
**Pull request by 13h3r**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -117,14 +127,48 @@ Connect a Consumer to Producer:
...
       db.update(msg.value).map(_ => msg.committableOffset)
     }
     .batch(max = 10, first => CommittableOffsetBatch.empty.updated(first)) { (batch, elem) =>
       batch.updated(elem)
     }
-    .mapAsync(1)(_.commit())
+    .mapAsync(1)(_.commitScaladsl())
+```
+
+Reusable kafka consumer:
```
Comment (ktoso) [inline README.md]: Needs a newline between text and code to render properly?

Comment (13h3r) [inline README.md]: Hmmm. I can not find any problems in browser - https://github.com/13h3r/reactive-kafka/blob/5a2d37ebe898009309a79e14d9134624c7c16537/README.md

File: README.md [hunk 2]
```diff
@@ -9,18 +9,30 @@ If you have questions or are working on a pull request or just curious, please f
...
+This version of `akka-stream-kafka` depends on Akka 2.4.7, Scala 2.11.8.
 
 Available at Maven Central for Scala 2.11:
 
 ````scala
-libraryDependencies += "com.typesafe.akka" %% "akka-stream-kafka" % "0.11-M3"
+libraryDependencies += "com.typesafe.akka" %% "akka-stream-kafka" % "0.11-M4"
 ````
 
+## Changes
```
Comment (patriknw) [inline README.md]: isn't this wrong heading level?

Comment (13h3r) [inline README.md]: Which one you suggest?

Comment (patriknw) [inline README.md]: it should be a sub-level of `## New API` so `### Changes`

File: README.md [hunk 3]
```diff
@@ -9,18 +9,30 @@ If you have questions or are working on a pull request or just curious, please f
...
+
+### 0.11-M4
```
Comment (patriknw) [inline README.md]: and this becomes `#### 0.11-M4`

File: README.md (hunk 2 above)
Comment (13h3r) [inline README.md]: ok

File: README.md (hunk 3 above)
Comment (13h3r) [inline README.md]: ok

**Pull request merged**